webapp/channels/src/components/widgets/modals/components/modal_header.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {CloseIcon} from '@mattermost/compass-icons/components';
+import './modal_header.scss';
+
+type Props = {
+    id: string;
+    title: string;
+    subtitle: string;
+    handleClose?: (e: React.MouseEvent) => void;
+}
+
+function ModalHeader({id, title, subtitle, handleClose}: Props) {
```

Comment:
Just to confirm, is it intentional that we're not using React Bootstrap's `Modal.Header` component here? This seems like a good way to start moving us away from React Bootstrap more and more

Reply:
Yeah, the purpose was to create our own header and remove the bootstrap modal in phases.